### PR TITLE
changed return type in calculate method

### PR DESCRIPTION
**Pull request by vmlysenkov**
@khodya please, check upgrades. I created new class with fields "days" and "hours" instead of ArrayList. I also changed names in tests, because I've completed testing course and old names were bad practice.

### CODE REVIEW

File: src/test/java/DetermineAmountOfTimeBeforeOrAfterCourseCompletionTest.java [hunk 1]
```diff
@@ -33,93 +33,79 @@ public void shouldDetermineEndDateWhenCourseEndsDuringWorkingDay() {
     }
 
     @Test
-    public void shouldCalculateDaysAndHoursAfterCourseCompletionUsingLocalDate1() {
```
Comment (khodya) [inline src/test/java/DetermineAmountOfTimeBeforeOrAfterCourseCompletionTest.java:36]: i don't like test renamings and new class are in the same commit

File: src/main/java/DurationBeforeOrAfterCourseCompletion.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import java.util.Objects;
+
+public class DurationBeforeOrAfterCourseCompletion {
```
Comment (khodya) [inline src/main/java/DurationBeforeOrAfterCourseCompletion.java:3]: would you consider to use Lombok or AutoValue libraries?

**Review by khodya — APPROVED** [review]

File: src/test/java/DetermineAmountOfTimeBeforeOrAfterCourseCompletionTest.java (hunk 1 above)
Comment (khodya) [reply src/test/java/DetermineAmountOfTimeBeforeOrAfterCourseCompletionTest.java:36]: never mind, it is related change

Comment (khodya) [reply src/test/java/DetermineAmountOfTimeBeforeOrAfterCourseCompletionTest.java:36]: no

File: src/main/java/DurationBeforeOrAfterCourseCompletion.java (hunk 2 above)
Comment (vmlysenkov) [reply src/main/java/DurationBeforeOrAfterCourseCompletion.java:3]: I haven't come across this AutoValue library yet.
From Lombok I know only @Value library, but it's not appropriate here.

Comment (khodya) [reply src/main/java/DurationBeforeOrAfterCourseCompletion.java:3]: ok, approved

**Pull request merged**